recipes/rethinkdb-python/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "rethinkdb-python" %}
+{% set version = "2.4.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/rethinkdb/rethinkdb-{{ version }}.tar.gz
+  sha256: 55ecf7447dc11fec0e2ce9aac4ba4c0b57fdc5c9c57d77ee673d59d245c74666
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [win]
```

Comment:
Why is this skipped? It looks like this could be a noarch package.

Reply:
The upstream package unfortunately does not support Windows (https://github.com/rethinkdb/rethinkdb-python/issues/217). I initially tried making it noarch, but this failed due to binary files and other issues. This PR doesn't include a detailed history of these failures, but I can reproduce them if needed.

Reply:
Sounds good to me. It's done in [77845af](https://github.com/conda-forge/staged-recipes/pull/28348/commits/77845af3812cc24175b0243e3b9199d3a745439c)!

Reply:
Hm, now win build fails with:

```
During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\a\1\s\.ci_support\build_all.py", line 356, in <module>
    build_all(os.path.join(root_dir, "recipes"), args.arch)
  File "D:\a\1\s\.ci_support\build_all.py", line 185, in build_all
    build_folders(recipes_dir, folders, arch, channel_urls)
  File "D:\a\1\s\.ci_support\build_all.py", line 248, in build_folders
    conda_build.api.build([recipe], config=get_config(arch, channel_urls))
  File "D:\Miniforge\Lib\site-packages\conda_build\api.py", line 209, in build
    return build_tree(
           ^^^^^^^^^^^
  File "D:\Miniforge\Lib\site-packages\conda_build\build.py", line 3655, in build_tree
    packages_from_this = build(
                         ^^^^^^
  File "D:\Miniforge\Lib\site-packages\conda_build\build.py", line 2762, in build
    newly_built_packages = bundlers[pkg_type](
                           ^^^^^^^^^^^^^^^^^^^
  File "D:\Miniforge\Lib\site-packages\conda_build\build.py", line 1871, in bundle_conda
    files = post_process_files(metadata, initial_files)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Miniforge\Lib\site-packages\conda_build\build.py", line 1666, in post_process_files
    noarch_python.populate_files(
  File "D:\Miniforge\Lib\site-packages\conda_build\noarch_python.py", line 95, in populate_files
    handle_file(f, d, prefix)
  File "D:\Miniforge\Lib\site-packages\conda_build\noarch_python.py", line 76, in handle_file
    fn = rewrite_script(fn, prefix)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Miniforge\Lib\site-packages\conda_build\noarch_python.py", line 31, in rewrite_script
    raise CondaBuildUserError(f"Noarch package contains binary script: {fn}")
conda_build.exceptions.CondaBuildUserError: Noarch package contains binary script: rethinkdb-export.exe
```

Not sure if this means this package cannot be `noarch` or if there is a resolution that I am not aware of.

Reply:
You need to specify the Python entry_points as seen in https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points